src/layer/heatmap/HeatMapLayer.js

```diff
@@ -0,0 +1,410 @@
+define([
```

Comment:
This file is missing the copyright header.

Reply:
Done